spec/models/institute_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe Institute, type: :model do
+  
+  subject { institute }
+  context 'with valid attributes' do
+    let!(:institute) { build(:institute, name: "institute1") }
+    
+    it { is_expected.to validate_presence_of(:name) }
+
+    it "is valid" do
+      subject
+      expect(institute).to be_valid
+    end
+  end
+
+  context "with invalid attributes" do
+      let!(:institute) { build(:institute, name: nil) }
+
+      it "is not valid" do
+        institute.valid?
+        expect(institute.errors[:name]).to include("can't be blank")
```

Comment:
do really need to check the error message here.
The presence errors on a models attribute are a core part of ActiveRecord and are already tested as part of ActiveRecords specs.  
In this case .. just a test for is not valid would be enough .. see below